webpack-widget.js

```diff
@@ -5,38 +5,56 @@ const CleanPlugin       = require('clean-webpack-plugin')
 const CopyPlugin        = require('copy-webpack-plugin')
 const ExtractTextPlugin = require('extract-text-webpack-plugin')
 const ZipPlugin         = require('zip-webpack-plugin')
-
-const MateriaDevServer = require('./express');
-
-// Default Materia Widget Config
-const defaultCfg = {
-	cleanName: '',
-	srcPath: path.join(process.cwd(), 'src'),
-	outputPath: path.join(process.cwd(), 'build'),
-	demoPath: 'demo.json',
-	installPath: 'install.yaml',
-	iconsPath: '_icons',
-	scorePath: '_score/',
-	screenshotsPath: '_screen-shots/',
-	assetsPath: 'assets/',
-	preCopy: []
-}
+const MateriaDevServer  = require('./express');
 
 // creators and players may reference materia core files directly
 // To do so rather than hard-coding the actual location of those files
 //the build process will replace those references with the current relative paths to those files
-
 const packagedJSPath = 'src=\\"../../../js/$3\\"'
 const devServerJSPath = 'src=\\"/mdk/assets/js/$3\\"'
 const isRunningDevServer = process.argv.find((v) => {return v.includes('webpack-dev-server')} )
 const replaceTarget = isRunningDevServer ? devServerJSPath : packagedJSPath
+const srcPath = path.join(process.cwd(), 'src') + path.sep
+const outputPath = path.join(process.cwd(), 'build') + path.sep
+const browserList = [
+	'Explorer >= 11',
+	'last 3 Chrome versions',
+	'last 3 ChromeAndroid versions',
+	'last 3 Android versions',
+	'last 3 Firefox versions',
+	'last 3 FirefoxAndroid versions',
+	'last 3 iOS versions',
+	'last 3 Safari versions',
+	'last 3 Edge versions'
+]
+
+const copyIgnore = [
+	'.gitkeep'
+]
 
 const materiaJSReplacements = [
 	{ search: /src=(\\?("|')?)(materia.enginecore.js)(\\?("|')?)/g,  replace: replaceTarget },
 	{ search: /src=(\\?("|')?)(materia.scorecore.js)(\\?("|')?)/g,   replace: replaceTarget },
 	{ search: /src=(\\?("|')?)(materia.creatorcore.js)(\\?("|')?)/g, replace: replaceTarget },
 ];
 
+const getDefaultEntries = () => ({
```

Comment:
New Feature: getDefaultEntries can now be used to create a copy of the entries used in `getLegacyWidgetBuildConfig`.  This can be used to re-use or override entries.